include/z64.h

```diff
@@ -34,6 +34,7 @@
 #include "z64item.h"
 #include "z64light.h"
 #include "z64math.h"
+#include "sys_matrix.h"
 #include "z64object.h"
 #include "z64ocarina.h"
 #include "z64player.h"
```

Comment:
Seems like the different categories mean something here: the general headers are all in the bottom one. This could be called z64matrix.h, but I decided not to, and put it in alphabetical order based on "matrix" instead.